homeassistant/components/media_player/universal.py

```diff
@@ -180,14 +181,19 @@ def _child_attr(self, attr_name):
     def _call_service(self, service_name, service_data=None,
                       allow_override=False):
         """Call either a specified or active child's service."""
-        if allow_override and service_name in self._cmds:
-            call_from_config(
-                self.hass, self._cmds[service_name], blocking=True)
-            return
-
```

Comment:
I'm not entirely sure why this was called in such a manner rather than with `self.hass.services.call`. I tried asking on gitter but no response so here we are. 🤕 

Looks like one test is falling around here, but the config used in the test does not make sense to me. The test configures a command that lacks an entity ID. How is it supposed to know what to operate on?